Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1413,16 +1421,41 @@ def parseArgs(args):
     if len(axisLimits) != len(options.locargs):
         parser.error("Specified multiple limits for the same axis")
 
+    if axisLimits and options.named_instances:
+        parser.error("Specified limits and --fvar")
+
     return (infile, axisLimits, options)
 
 
 def main(args=None):
     """Partially instantiate a variable font."""
     infile, axisLimits, options = parseArgs(args)
-    log.info("Restricting axes: %s", axisLimits)
 
     log.info("Loading variable font")
     varfont = TTFont(infile)
+    if options.named_instances:
+        for instance in varfont["fvar"].instances:
```

Comment:
maybe move this whole for loop to its own function to `instantiateVariableFontNamedInstances`